### PR TITLE
[MPI] fix skipping of tests if Metis is not available

### PR DESCRIPTION
**Pull request by philbucher**
**Description**
See title

I also updated a test that didn't use the new functionalities yet

### CODE REVIEW

File: kratos/python_scripts/testing/utilities.py [hunk 1]
```diff
@@ -59,7 +60,6 @@ def ReadDistributedModelPart(mdpa_file_name, model_part, importer_settings):
         mdpa_file_name (str): Name of the mdpa file (without ".mdpa" extension)
         model_part (Kratos.ModelPart): ModelPart to be filled
     """
-    KratosUnittest.skipIfApplicationsNotAvailable("MetisApplication")
```
Comment (philbucher) [inline kratos/python_scripts/testing/utilities.py:62]: this doesn't work, it needs to be a decorator

File: kratos/mpi/tests/test_mpi_model_part.py [hunk 2]
```diff
@@ -1,31 +1,12 @@
 import KratosMultiphysics
 import KratosMultiphysics.KratosUnittest as KratosUnittest
+from KratosMultiphysics.testing.utilities import ReadDistributedModelPart
 
 import os
 
 def GetFilePath(fileName):
     return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
 
-def ReadDistributedModelPart(model_part, mdpa_file_name):
```
Comment (roigcarlo) [inline kratos/mpi/tests/test_mpi_model_part.py:9]: Ping @KratosMultiphysics/altair this might break some of your tests if I recall correctly, be careful

Comment (philbucher) [reply kratos/mpi/tests/test_mpi_model_part.py:9]: @KratosMultiphysics/altair if this breaks your tests then please use the same fct I use in this PR (it does the same)

**Comment by philbucher** [comment]
can we merge this? It is blocking the Intel compiler PR :/

File: kratos/mpi/tests/test_mpi_model_part.py (hunk 2 above)
Comment (ddiezrod) [reply kratos/mpi/tests/test_mpi_model_part.py:9]: I think most tests should be fine. Thanks for pinging! ;)

**Review by pooyan-dadvand — APPROVED** [review]
👍

**Pull request merged**